sktime/transformations/panel/tsfresh.py

```diff
@@ -127,15 +128,101 @@ def _get_extraction_params(self):
             fc_parameters = self.default_fc_parameters
         extraction_params["default_fc_parameters"] = fc_parameters
 
+        # creates mapping from kind names to fc_parameter objects
+        if self.kind_to_fc_parameters is not None:
+            self.kind_to_fc_parameters_ = from_columns(self.kind_to_fc_parameters)
+        else:
+            self.kind_to_fc_parameters_ = self.kind_to_fc_parameters
+        extraction_params["kind_to_fc_parameters"] = self.kind_to_fc_parameters_
         return extraction_params
 
 
 class TSFreshFeatureExtractor(_TSFreshFeatureExtractor):
     """Transformer for extracting time series features.
 
+    Parameters
+    ----------
+    default_fc_parameters : string, FCParameters object or None
+        Specifies pre-defined feature sets to be extracted
+        If string should be in ["minimal", "efficient", "comprehensive"]
+        See [3] for more details.
```

Comment:
The default value should be included, i.e. `default_fc_parameters : string, FCParameters object or None, default=None`. In this case it would be good to include what None actually does as well. 

I wouldn't be afraid to take directly from the `tsfresh` docstrings for these if they are available.